developer_guide/migration/index.rst

```diff
@@ -1,12 +1,84 @@
 Migration
 =========
 
-There are upgrade documentations from a minor version to the next one.
+There are upgrade documentations from a patch or minor version to the next one.
 
 We assume that you're using a project installed from a community standard (:doc:`/developer_guide/installation/installation_workstation`).
 
-Community Edition
------------------
+Patch Version
+-------------
+
+We release regularly patch versions, these versions provide bug fixes.
```

Comment:
We frequently release patch versions, in order to provides security and bug fixes.